napari/utils/_testsupport.py

```diff
@@ -210,7 +210,7 @@ def make_napari_viewer(
 
     viewers: WeakSet[Viewer] = WeakSet()
 
-    # may be overridden by using `make_napari_viewer(strict=True)`
+    # may be overridden by using `make_napari_viewer(strict_qt=True)`
```

Comment:
I think that this comment should document `"raise"` value also

Reply:
Yes I can do that but what about just saying it can be overridden by parameter `strict_qt` and they can read the docstring which gives all the details and prevents is from repeating ourself?

Reply:
ok

Reply:
Thanks, updated the comment so it refers to the parameter.